homeassistant/components/jewish_calendar/quality_scale.yaml

```diff
@@ -0,0 +1,92 @@
+rules:
```

Comment:
`set_language(self.data.language)` why do we do this in the entity?

Reply:
This method sets the language of the state returned by the hdate library by setting a context manager.
This is instead of the previous way, where we were passing around language values to the various classes making it a mess. (Especially as the language is just a representation and is irrelevant to the calculations).
If you have a better suggestion where to do this, I'd be happy to hear.

Reply:
But why do we do that for each entity